src/components/UniversalNavbarExpanded/CtaButton.js

```diff
@@ -15,10 +15,11 @@ import styles from './CtaButton.module.scss'
  * @param {string} href - URL for the button to link to
  * @param {function} trackingFunction - Analytics function run when CTA Button is clicked
  * @param {boolean} hideOnMobile - Hide the CTA on phone only
+ * @param {string} copy - Copy text for the button
  *
  * @return {JSX.Element}
  */
-const CtaButton = ({ href, trackingFunction, hideOnMobile }) => {
+const CtaButton = ({ href, trackingFunction, hideOnMobile, copy }) => {
```

Comment:
@coleEthos why do we call this new property `copy`? Can we name it something more self-explanatory for the button text usage?

Reply:
@ChirashiSushi there are places where we interchange `copy` with `text` but I agree this could be better named. Will update to `title` to match the schema of the links provided to the component.